### ISSUE TITLE
Cannot use beta version on android

### ISSUE THREAD

**Issue by sebazh** [syanbo/react-native-agora#223]
Hi guys. I’m trying to use the latest 3.0.1-beta.5 version of react-native-agora, but got the following error on android when call 
```
this.engine = await RtcEngine.create(appConfig.agoraAppId); 
```
Please help me to understand what’s wrong

Logcat info:
`E/linker: "/data/app/test.app.stg-zQaTNAd2eIDqbpVBgsopVQ==/lib/x86/libagora-rtc-sdk.so" has text relocations (https://android.googlesource.com/platform/bionic/+/master/android-changes-for-ndk-developers.md#Text-Relocations-Enforced-for-API-level-23)
 E/unknown:ReactNative: Exception in native call
    java.lang.RuntimeException: Could not invoke RCTAgoraRtcEngineModule.create`

**Comment by LichKing-2234** [comment]
@sebazh this is a bug from native SDK, can you try it on physical device? I will fix it next version.

**Comment by sebazh** [comment]
@LichKing-2234 Yes, on real device it works. Thanks. Will wait for your next release

**Comment by LichKing-2234** [comment]
fixed in 3.0.1-rc.1
